src/main/app/src/constants/keys.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const info = [{
+      airtable_api_key_helpPage: "keyvEcGAXuzDAVmVb",
+      clientID:"298755462-7fnqhho2db1b0mb5i6o7irgg8v7om76g.apps.googleusercontent.com",
+      Airtable_URL:"https://api.airtable.com/v0/appun5As9N6CFqu4H/Resources?api_key=keyvEcGAXuzDAVmVb"
+
+}];
+
+export default info;
```

Comment:
new line at EOF.

Reply:
Will address in next pull request.

Reply:
For small changes like this we should address them in the same pull request.

Larger refactors can wait to the next pull request.

Reply:
I think Bhargavi has added the fix to next pull request, so I am going to approve this one.

Keep in mind that you should address small changes in the same pull request, so for all pull requests later, make sure you create new branches so that you can modify it

Reply:
Sure, got it. Thank you so much!